### PR TITLE
tthsum: init at 1.3.2

### PR DESCRIPTION
**Pull request by ebzzry**

### CODE REVIEW

File: pkgs/applications/misc/tthsum/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/man/man1
+    cp share/tthsum.1.gz $out/share/man/man1
+    cp obj-unix/tthsum $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "tthsum is an md5sum-alike program that works with Tiger/THEX hashes";
```
Comment (pSub) [inline pkgs/applications/misc/tthsum/default.nix]: Do not repeat the package name in `description`. See http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

**Comment by pSub** [comment]
Otherwise the PR looks good to me.

File: pkgs/applications/misc/tthsum/default.nix (hunk 1 above)
Comment (ebzzry) [inline pkgs/applications/misc/tthsum/default.nix]: @pSub fixed

**Comment by pSub** [comment]
Thanks.

**Pull request merged**